### ISSUE TITLE
Cystic Fibrosis-related diabetes

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#7946]
**Preferred term label:**
Cystic Fibrosis-related diabetes

**Synonyms**
CF-related diabetes

**Definition (free text, please give PubMed ID)**
CF-related diabetes has distinctive
peculiarities that make it different from type 1 and type 2:
it originates from reduced secretion of insulin, but is also
partially due to insulin resistance, particularly during acute
pulmonary exacerbations  PMID: 27709245 

 Cystic Fibrosis-Related Diabetes Mellitus (CFRD) results from ongoing 
obstructive damage to the pancreas from thick secretions, which in turn results in fatty 
infiltration of the pancreas and islet cell destruction. (16) CFRD typically presents after 
the first decade, and occurs in up to 20% of adolescents and at least 50% of adults with 
CF. (57) CFRD is typically diagnosed in patients with pancreatic insufficiency and has 
been associated with increased morbidity and mortality through worse nutritional status and 
decreased lung function PMID: 33526571

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Cystic fibrosis OMIM:219700

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@carolina-f 
The HPO avoids encoding pathophysiology, because if it did, then there would be a profusion of terms such as disease X-related HPOterm Y. Also, diabetes itself is a bundled term that is itself not an ideal HPO term. I would suggest that if we want to capture the phenotypes of CF related DM, then individual HPO terms e used (and potentially developed if needed).
